app/src/main/res/layout/fit_fragment.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is the reason to add legacy support library? I suppose, there are many other ways to avoid it. If you really need to use it for some specific reason, this reason should be written as a comment here and in import as well.